contracts/ClaimZap.sol

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.7;
+
+import "./helper/MathUtil.sol";
+import "./Interfaces/IBasicRewards.sol";
+import "./Interfaces/IVeTokenRewards.sol";
+import "./Interfaces/IVeAssetDeposit.sol";
+import "./Interfaces/ISwapExchange.sol";
+import "./Interfaces/IVe3dLocker.sol";
```

Comment:
our interfaces not organize based on contracts , but based in same group of functionality , so I recommend to use IReward for IBasicRewards, IVeTokenRewards ,